spec.bs

```diff
@@ -91,7 +94,7 @@ Each item must be either a {{PAHistogramContribution}} or a
 {{PAExtendedHistogramContribution}}.
 
 Note: The steps to process the [=contributions cache=] are defined separately for
-each {{WorkletGlobalScope}}.
+each [=context type=].
```

Comment:
Context type is fledge or shared storage, isn't this per global scope?

Reply:
What I'm trying to convey here is that there's a single algorithm defined for all of FLEDGE. (Separately, I have to transition this spec to per-auction rather than per-global scope processing.) Lmk if you have any ideas to reword

Reply:
Ah yes, I misread this as the definition. Sorry for the noise.